packages/lab/src/date-input/DateInputSingle.tsx

```diff
@@ -160,17 +161,17 @@ export const DateInputSingle = forwardRef<
       defaultDate,
       onDateChange,
       value: valueProp,
-      defaultValue = "",
+      locale,
+      format = "DD MMM YYYY",
+      defaultValue = dateAdapter.format(undefined, format, locale),
       onChange,
       onClick,
       emptyReadOnlyMarker = "—",
       endAdornment,
-      format = "DD MMM YYYY",
       inputProps = {},
       inputRef: inputRefProp = null,
-      locale,
       parse: parseProp,
-      placeholder = "dd mmm yyyy",
+      placeholder = format.toLowerCase(),
```

Comment:
lower case and upper case in date formatter are different. A few suggestions
- Remove, leave empty (no other Salt component sets this)
- Set as generic text, e.g. "Set Date", "Start Date", etc.
- Or leave as `format`

Reply:
@ivan-calderon / @jake-costa / @honey-chang  one for you to opine on..
Feels like this falls into the formatting pattern, an overall guide on recommended formats?
Simple change to make once we agree on what it should be (compare with UITK?)